.bazelrc

```diff
@@ -3,6 +3,7 @@ common --enable_platform_specific_config
 # Shared configuration flags to build and test Bazel on RBE.
 build:remote_shared --remote_instance_name=projects/bazel-untrusted/instances/default_instance
 build:remote_shared --remote_executor=grpcs://remotebuildexecution.googleapis.com
+build:remote_shared --remote_download_minimal
```

Comment:
Do we want `--remote_download_toplevel`? Since most of the time when I build Bazel, I still want the Bazel binary be downloaded.

Reply:
Yeah, that's fair. Amended.